signer/core/api.go

```diff
@@ -552,7 +553,11 @@ func (api *SignerAPI) SignTransaction(ctx context.Context, args apitypes.SendTxA
 	}
 	// If we are in 'rejectMode', then reject rather than show the user warnings
 	if api.rejectMode {
-		if err := msgs.GetWarnings(); err != nil {
+		err := msgs.GetWarnings()
+		if err != nil {
+			if strings.Contains(err.Error(), "ABI signature could not be found") {
+				return nil, fmt.Errorf("%s. Please use '--advanced' mode or provide the 'selector' argument to force the signing", err.Error())
+			}
```

Comment:
Perhaps just keep it simple? 
```suggestion
			log.Info("Signing aborted due to warnings. In order to continue despite warnings, please use the flag '--advanced'.")
```